zebra/zebra_vxlan.c

```diff
@@ -5032,6 +5032,13 @@ int zebra_vxlan_if_update(struct interface *ifp, uint16_t chgflags)
 			return 0;
 		}
 
+		if ((chgflags & ZEBRA_VXLIF_MASTER_MAC_CHANGE)
+		    && if_is_operative(ifp) && is_l3vni_oper_up(zl3vni)) {
+			zebra_vxlan_process_l3vni_oper_down(zl3vni);
```

Comment:
if MAC change happens when l3vni is down, how is that handled ?

Reply:
if l3vni is down, then evpn contexts are already flushed.
so the sync will happen when l3vni will become up.
for instance, when bridge interface will go up.